tools/cli/namespaceCommands.go

```diff
@@ -208,6 +209,14 @@ func (d *namespaceCLIImpl) UpdateNamespace(c *cli.Context) {
 		if c.IsSet(FlagOwnerEmail) {
 			ownerEmail = c.String(FlagOwnerEmail)
 		}
+		if c.IsSet(FlagState) {
+			stateStr := c.String(FlagState)
+			if stateInt, ok := enumspb.NamespaceState_value[stateStr]; !ok {
+				ErrorAndExit(fmt.Sprintf("Unknown namespace state: %s. Supported states: [Registered, Deprecated, Deleted, Handover].", stateStr), nil)
```

Comment:
include actual namespace name in log message?

Reply:
This is not logging. This is for tctl and the message is print to console when user try to do invalid things. The namespace is what user supplied to the command.